flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/source/TestMetadataTableReadableMetrics.java

```diff
@@ -217,27 +217,27 @@ public void testPrimitiveColumns() throws Exception {
 
     Row binaryCol =
         Row.of(
-            52L,
+            55L,
```

Comment:
Why the size is growing here (I mean in the test) ? 
Should we have two tests ?

Reply:
See https://github.com/apache/iceberg/pull/11502#discussion_r1835516115 for the reason. What's the suggestion for the second test?